packages/ethereum-contracts/scripts/getConfig.js

```diff
@@ -6,7 +6,7 @@ const SuperfluidSDK = require("@superfluid-finance/js-sdk");
  */
 
 const DEFAULT_CONFIGS = {
-    8645: {
+    4447: {
```

Comment:
How was this port selected?

Reply:
this the local ganache chainId, i used the port number by mistake